app/design/frontend/base/default/template/securesubmit/form.phtml

```diff
@@ -137,8 +137,9 @@ SecureSubmitMagento.init({
 
 <?php if ($_loggedIn && $allow_card_saving && count($customerStoredCards)): ?>
 <script type="text/javascript">
-(".saved-card-selector input").on('selected', function () {
-    (this).parent().toggleClass('selected');
- });
+var secureSubmitsavedcardselector = document.querySelector('.saved-card-selector input');
+secureSubmitsavedcardselector.addEventListener('click', function(){
+    secureSubmitsavedcardselector.parentNode.classList.toggle('selected')
+}, false);
```

Comment:
@ttnnkkrr lines 141-143 should be wrapped in an if statement to ensure `querySelector` returns a valid value. jQuery and Prototype handle this checking for you, so in plain Javascript, you'll need to ensure this happens to prevent a possible reference error.

Also, `classList` is not supported in all versions of IE, so you'll want to use Prototype to facilitate this (if possible) or manually loop through the node's active classes to toggle `.selected`.

Line 142 should also end with a semi-colon.